app/src/main/java/com/babylon/wallet/android/presentation/account/AccountScreen.kt

```diff
@@ -257,7 +264,7 @@ fun AssetsContent(
         color = RadixTheme.colors.gray5
     ) {
         val accountAddress = remember(state.accountWithAssets) {
-            state.accountWithAssets?.account?.address.orEmpty()
+            state.accountWithAssets?.account?.let { AccountAddress.init(it.address) }
```

Comment:
maybe AccountWithAssets should expose property/function to get typed address instead of doing it in composable?

Reply:
This will be replaced with sargon's profile later which will expose an `AccountAddress` directly. So I had to bridge it like this for now.